Apps/Database/src/Context/GatewayDbContext.cs

```diff
@@ -70,6 +70,7 @@ public GatewayDbContext(DbContextOptions<GatewayDbContext> options)
         public DbSet<Comment> Comment { get; set; } = null!;
         public DbSet<Communication> Communication { get; set; } = null!;
         public DbSet<CommunicationEmail> CommunicationEmail { get; set; } = null!;
+        public DbSet<Ratings> Ratings { get; set; } = null!;
```

Comment:
I think we are using table name without "s". Ratings --> Rating

Reply:
singular vs plural, sure